message/batch.go

```diff
@@ -80,52 +82,18 @@ func (m *Batch) Flags() primitive.QueryFlag {
 	return flags
 }
 
-// Clone performs a deep copy of this batch message object
-func (m *Batch) Clone() Message {
-	var newBatchChildren []*BatchChild
-	for _, child := range m.Children {
-		newBatchChildren = append(newBatchChildren, child.Clone())
-	}
-
-	return &Batch{
-		Type:              m.Type,
-		Children:          newBatchChildren,
-		Consistency:       m.Consistency,
-		SerialConsistency: m.SerialConsistency.Clone(),
-		DefaultTimestamp:  m.DefaultTimestamp.Clone(),
-		Keyspace:          m.Keyspace,
-		NowInSeconds:      m.NowInSeconds.Clone(),
-	}
-}
-
+// BatchChild represents a BATCH child statement.
+// +k8s:deepcopy-gen=true
 type BatchChild struct {
-	// Either a string or []byte; the former should be the CQL statement to execute, the latter the prepared id of
-	// the statement to execute.
-	QueryOrId interface{}
+	// The CQL statement to execute. Exactly one of Query or Id must be present, never both.
+	Query string
```

Comment:
This change was induced by the generator because it does not support interface{} types. But I think this is a good change anyways, I don't even know why I used a single field for both Query and Id while it's so much simpler to use 2 distinct fields.

Reply:
Agree 👍